src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] The App.js component should be class-based, Kindly update it 💯

![image](https://user-images.githubusercontent.com/45231276/189803548-0d22d00e-1451-4e1a-bb3a-b41a45b031b9.png)

- [ ] Make sure to delete all the boilerplate from CRA (text, images, styles)., So I suggest deleting `index.css`, `App.test.js`, `setupTests.js`, and `reportWebVitals.js` files 👌